pakyow-presenter/lib/presenter/view.rb

```diff
@@ -492,13 +492,22 @@ def bind_attributes_to_doc(attrs, doc)
             attr  = attr.to_s
             attrs = Attributes.new(doc)
 
+            # TODO With the introduction of parts, values should only be
```

Comment:
Correct. Previously this logic allowed for modifying the current value:

```ruby
binding :foo do
  { href: -> (href) { do_something_to_current_href_value } }
end
```

What that would look like now is:

```ruby
binding :foo do
  part :href do |href|
    do_something_to_current_href_value
  end
end
```